### PR TITLE
Use query features

### PR DESCRIPTION
**Pull request by kkraune**
- also remove useless summary feature attribute(namespace)

### CODE REVIEW

File: src/main/application/schemas/doc.sd [hunk 1]
```diff
@@ -82,23 +82,24 @@ schema doc {
 
     rank-profile documentation inherits default {
         rank-properties {
-            $titleWeight: 2.0
-            $contentsWeight: 1.0
-            $headerWeight: 1.0
-            $pathWeight: 1.0
+            query(titleWeight): 2.0
+            query(contentsWeight): 1.0
```
Comment (geirst) [inline src/main/application/schemas/doc.sd:86]:
```suggestion
            query(contentWeight): 1.0
```

Comment (geirst) [reply src/main/application/schemas/doc.sd:86]: To be aligned with the name of the field and what is used in another rank profile. Also the expression and summary-features must be changed.

**Review by geirst — APPROVED** [review]
👍

File: src/main/application/schemas/doc.sd (hunk 1 above)
Comment (kkraune) [reply src/main/application/schemas/doc.sd:86]: good point, will do

**Pull request merged**